ios/Video/RCTVideo.swift

```diff
@@ -1620,6 +1620,12 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
         }
     }
 
+    @objc
+    func getCurrentPlaybackTime(_ resolve: @escaping RCTPromiseResolveBlock) {
+        let currentTime = RCTVideoUtils.getCurrentTime(playerItem: _playerItem)
+        resolve(currentTime)
```

Comment:
it would be great to match android behavior: if playerItem is nil, reject the promise.